plugins/node.d/hddtemp_smartctl

```diff
@@ -219,11 +219,14 @@ if (defined $ARGV[0]) {
     print "graph_category sensors\n";
     print "graph_info This graph shows the temperature in degrees Celsius of the hard drives in the machine.\n";
     foreach (@drives) {
+	my $d = clean_fieldname($_);
         my @dirs = splitdir($_);
-        print clean_fieldname($_) . ".label " . $dirs[-1] . "\n";
-        print clean_fieldname($_) . ".max 100\n";
-        print clean_fieldname($_) . ".warning 57\n";
-        print clean_fieldname($_) . ".critical 60\n";
+        print $d . ".label " . $dirs[-1] . "\n";
```

Comment:
Maybe you could align the indentation here? (including the `splitdir` line above)

Reply:
I've spaced it (I'll need to reconstruct my .emacs file for official work..). I belive the shortest path forward now is that you select a squash merge - and make the commit comment something like "Identify drives beyond the device file" - at least we can do this in the GitHubEnterprise I'm used to. I'm not allowed to merge this PR to master myself.

Thanks